src/BikeDataProject.Registrations.API/Controllers/WebRegistrationController.cs

```diff
@@ -79,7 +79,7 @@ public async Task<IActionResult> RegisterStrava(String code)
                     IsHistoryFetched = false
                 };
 
-                if (this._dbContext.Users.FirstOrDefault(u => (u.ProviderUser == user.ProviderUser) || (u.Provider == Constants.WebStravaUserAgent)) == null)
```

Comment:
|| bad

Reply:
You didn't even notice it
AHAH GET PRANKED